src/battle/partner/twink_714CF0.c

```diff
@@ -1,5 +1,12 @@
 #include "common.h"
 
-INCLUDE_ASM(s32, "battle/partner/twink_714CF0", func_80238000_714CF0);
-
+ApiStatus func_80238000_714CF0(ScriptInstance *arg0) {
```

Comment:
The signature for this func should be `ApiStatus func_80238000_714CF0(ScriptInstance* script, s32 isInitialCall)`